config/configauth/configauth.go

```diff
@@ -26,14 +26,14 @@ var (
 	errAuthenticatorNotFound = errors.New("authenticator not found")
 )
 
-// Authentication defines the auth settings for the receiver
+// Authentication defines the auth settings for the receiver.
 type Authentication struct {
 	// AuthenticatorName specifies the name of the extension to use in order to authenticate the incoming data point.
 	AuthenticatorName string `mapstructure:"authenticator"`
 }
 
-// GetServerAuthenticator attempts to select the appropriate from the list of extensions, based on the requested extension name.
-// If an authenticator is not found, an error is returned.
+// GetServerAuthenticator attempts to select the appropriate ServerAuthenticator from the list of extensions, based on the requested extension name.
+// If authenticator is not found, an error is returned.
```

Comment:
```suggestion
// If an authenticator is not found, an error is returned.
```
I think this is correct.  Alternately:

```suggestion
// If no authenticator is found, an error is returned.
```